getting-started-guide/observability.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: "Axiom for observability"
+description: "This page explains how Axiom helps you leverage timestamped event data for observability purposes."
+sidebarTitle: Observability
+---
+
+Axiom helps you leverage the power of timestamped event data. A common use case of event data is observability (o11y) in the field of software engineering. Observability means measuring the health and performance of apps and distributed systems. The goal of collecting this information is to monitor the status of apps and systems, improve their quality, and fix errors.
+
+Software engineers most often work with timestamped event data in the form of logs or metrics. However, Axiom believes that event data reflects a much broader range of interactions, crossing boundaries from engineering to product management, security, and beyond. For a more general explanation of event data in Axiom, see [Events](/getting-started-guide/event-data).
+
+## Types of event data in observability
+
+Traditionally, observability has been associated with three pillars, each effectively a specialized view of event data:
+
+- **Logs**: Logs record discrete events, such as error messages or access requests, typically associated with engineering or security.
+- **Traces**: Traces track the path of requests through a system, capturing each step’s duration. By linking related spans within a trace, developers can identify bottlenecks and dependencies.
+- **Metrics**: Metrics quantify state over time, recording data like CPU usage or user count at intervals. Product or engineering teams can then monitor and aggregate these values for performance insights.
+
+In Axiom, these observability elements are stored as event data, allowing for fine-grained, efficient tracking across all three pillars.
+
+## Metrics support
```

Comment:
Given the paragraph before, I wonder if we shouldn’t at least touch on Axiom’s coverage of logs and distributed traces. For example, we could point to other pages in the documentation that are most relevant for those kinds of event data? 

Otherwise, I jump immediately to the part we do _not_ fully support, with its limitations.